apps/condo/domains/acquiring/schema/fields/common.js

```diff
@@ -0,0 +1,11 @@
+const { Text } = require('@keystonejs/fields')
+
+const IMPORT_ID_FIELD = {
+    schemaDoc: 'Id of `acquiring object` which represents current item in base of external service. Used for internal needs of `acquiring integration service`',
+    type: Text,
+    isRequired: false,
+}
+
+module.exports = {
+    IMPORT_ID_FIELD,
+}
```

Comment:
We might want to move this field to common, since it is used in billing and acquiring model

Reply:
done 👍